app/uk/gov/hmrc/cbcr/services/AdminService.scala

```diff
@@ -65,80 +65,6 @@ class AdminService @Inject()(
   def displayAllDocRefId(docs: List[DocRefIdRecord]): ListDocRefIdRecord =
     ListDocRefIdRecord(docs)
 
```

Comment:
do we not want to delete the entire service?

Reply:
This bit of code is used in another controller for legitimate reasons. Do you think we should perhaps inline it?

Reply:
Ah OK, perhaps, no strong opinion either way

Reply:
Turned out this is an unused admin route and doesn't have `/admin/` in the URL. Deleted.